install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -122,9 +122,6 @@ spec:
               name: cilium-config
               optional: true
 {{- if .Values.global.hubble.enabled }}
-        # Hubble uses this group for its unix domain sockets.
-        - name: HUBBLE_GROUP_NAME
-          value: "cilium"
```

Comment:
Note: This has not been functional since f6e82ec8653288397d9c7af2f0565895ee7a95f4